internal/commands/pull/inputs.go

```diff
@@ -99,10 +99,6 @@ type from struct {
 func (i inputs) resolveFrom(ui terminal.UI, client realm.Client) (from, error) {
 	f := from{GroupID: i.Project}
 
-	if i.From == "" {
```

Comment:
this caused us to pull from empty apps sometimes (if not run from within a project directory), by removing this we simply "resolve the app" with an empty app filter (which just presents the user with a larger list of apps to select from)